HygrometerApp/Resources/Network/API.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  API.swift
+//  HygrometerApp
+//
+//  Created by hyeonseok on 2022/07/11.
+//
+
+import Foundation
+
+public struct API {
+    public init() {}
+    
+    
+    /// Add String
+    /// - Parameter stringArray: 여러개의 string 붙이기
+    /// - Returns: 배열로 들어온 string 붙여서 하나라 리턴
+    static public func makeString(With stringArray: [String]) -> String {
```

Comment:
`argument label`에 대문자가 들어가 있습니다.
```suggestion
    static public func makeString(with stringArray: [String]) -> String {
```